test/metabase/query_processor_test/native_test.clj

```diff
@@ -37,6 +38,34 @@
            (mt/native-query
             {:query "select name from users;"}))))))
 
+(deftest ^:parallel native-with-duplicate-column-names
+  (testing "Should be able to run native query referring a question referring a question (#25988)"
+    (mt/with-test-drivers (sql-jdbc.tu/sql-jdbc-drivers)
```

Comment:
Test is running against jdbc drivers, but the change affects `sql.qp`. I'm wondering if the change could not impact also bigquery (parent there is sql).

Reply:
great catch